.gitignore

```diff
@@ -2,3 +2,4 @@
 .Rhistory
 .RData
 inst/doc
+docs
```

Comment:
Avoid accidentally pushing the website; it builds on GH Actions and is published on the branch gh-pages.